components/General/index.tsx

```diff
@@ -23,32 +23,31 @@ export const Box = styled.div`
     padding: 12px;
 `;
 
-export const Button = styled.button`
+export const Button = styled.button<{ height?: 'medium' | 'small' | 'extra-small' }>`
```

Comment:
Can you add a default props here. Maybe medium?

Reply:
can do, but not giving it a prop would just be equivalent to not setting a height

Reply:
and my thought was that for any template component we'd avoid giving it preset attributes

Reply:
Thats a fair point. Can leave it